src/content/docs/distribute/Signing/windows.mdx

```diff
@@ -0,0 +1,278 @@
+---
+title: Windows Code Signing
+sidebar:
+  label: Windows
+  order: 2
+---
+
+Code signing your application lets users know that they downloaded the official executable of your app and not some 3rd party malware that poses as your app.
+While it is not required, it improves users' confidence in your app.
+
+This guide covers signing via OV (Organization Validated) certificates and Azure Key Vault. If you use any other signing mechanism not documented here,
+check out your certificate issuer documentation and refer to the [custom sign command](#custom-sign-command) section.
+
+## OV Certificates
+
+:::danger
+
+This guide only applies to OV code signing certificates acquired before June 1st 2023! For code signing with EV certificates and OV certificates received after that date please consult the documentation of your certificate issuer instead.
+
+:::
+
+### Prerequisites
+
+- Windows - you can likely use other platforms, but this tutorial uses Powershell native features.
+- A working Tauri application
+- Code signing certificate - you can acquire one of these on services listed in [Microsoft's docs]. There are likely additional authorities for non-EV certificates than included in that list, please compare them yourself and choose one at your own risk.
+  - Please make sure to get a **code signing** certificate, SSL certificates do not work!
+
+This guide assumes that you have a standard code signing certificate> If you have an EV certificate, which generally involves a hardware token, please follow your issuer's documentation instead.
+
+:::note
+
+If you sign the app with an EV Certificate, it'll receive an immediate reputation with Microsoft SmartScreen and won't show any warnings to users.
+
+If you opt for an OV Certificate, which is generally cheaper and available to individuals, Microsoft SmartScreen will still show a warning to users when they download the app. It might take some time until your certificate builds enough reputation. You may opt for [submitting your app] to Microsoft for manual review. Although not guaranteed, if the app does not contain any malicious code, Microsoft may grant additional reputation and potentially remove the warning for that specific uploaded file.
+
+:::
+
+### Getting Started
+
+There are a few things we have to do to get Windows prepared for code signing. This includes converting our certificate to a specific format, installing this certificate, and decoding the required information from the certificate.
+
+#### A. Convert your `.cer` to `.pfx`
+
+1. You will need the following:
```

Comment:
All of the numbered steps here could use the `<Steps>` feature for better user experience.